solidity/random-beacon/deploy/09_deploy_random_beacon_chaosnet.ts

```diff
@@ -33,3 +33,7 @@ const func: DeployFunction = async (hre: HardhatRuntimeEnvironment) => {
 export default func
 
 func.tags = ["RandomBeaconChaosnet"]
+
+// Only execute for chaosnet deployments.
```

Comment:
I think "chaosnet" tag will be a common point of confusion. Chaosnet is our rollout program (https://blog.threshold.network/tbtc-v2-hits-its-first-launch-milestone/) and the network we currently have deployed on mainnet is the Chaosnet.

In this PR, in `hardhat.config.ts` we flip it: everything but `mainnet` network is a Chaosnet.

What is the reason for introducing the `chaosnet` tag? I think we want to deploy `RandomBeaconChaosnet` for all networks.

Reply:
In the ecdsa deployment script we use the `chaosnet` tag as well.
Wouldn't it be even more confusing if in `random-beacon` we always execute `RandomBeaconChaosnet`-related scripts while in ecdsa we do that conditionally (when `chaosnet` tag is set)?

Reply:
Let's take a step back because I think we are confused about what we are trying to achieve. 

We have `RandomBeaconChaosnet` to deploy and to use it from `WalletRegistry` (proxy) contract.
- We do not want to start a revolution right now and rework a bunch of unit tests in `ecdsa/solidity` that work with a real `RandomBeacon`. So for `hardhat` network, we should use the real `RandomBeacon`. 
- I do not know about system tests (cc @dimpar) but I expect they use a real `RandomBeacon` as well given the `RandomBeaconChaosnet` was introduced quite recently.
- For Goerli, Mainnet, and local development (local geth) networks we want to use `RandomBeaconChaosnet`.

I do not think it hurts if we deploy `RandomBeaconChaosnet` for Hardhat network, it's just a few milliseconds. 

Regarding `solidity/ecdsa` contracts, I would make a rule that for all networks but `hardhat` we use the `RandomBeaconChaosnet`. Let's just check what to do about system tests.

Instead, we want to deploy `RandomBeaconChaosnet` and wire it up with `WalletRegistry`